var/spack/repos/builtin/packages/elpa/package.py

```diff
@@ -150,24 +160,31 @@ def configure_args(self):
         cuda_flag = "nvidia-gpu"
         if "+cuda" in spec:
             prefix = spec["cuda"].prefix
-            options.append("--enable-{0}".format(cuda_flag))
+            options.append(f"--enable-{cuda_flag}")
```

Comment:
shouldn't this be
```suggestion
            options.append(f"--enable-{cuda_flag}{kernels}"})
```

Reply:
It should, but unfortunately it does not work. See https://github.com/marekandreas/elpa/issues/55.

Reply:
Yeah, probably worth tracking it into the recipe with a link to the issue 😉

Reply:
```suggestion
            # Can't yet be changed to the new option --enable-nvidia-gpu-kernels
            # https://github.com/marekandreas/elpa/issues/55
            options.append(f"--enable-{cuda_flag}")
```

Reply:
Not asking for a change, just asking for my understanding:

Could the new `-kernels` suffix be used if `~gpu_streams`? I.e. you only get the configure error when using `--enable-nvidia-gpu-kernels` and `--enable-gpu-streams=nvidia` together?

Reply:
Yes, precisely. If `--enable-gpu-streams=nvidia`, then it checks that GPU support is active too on the old variable. Otherwise the new variables can be used. But I thought it would be better to do the change in one go (since it's deprecated, not yet removed), instead of introducing extra dependencies on variants. Do you prefer to change it? I don't mind either way.

Reply:
No, all good for me as well!